src/app/lib/views/file_selector.js

```diff
@@ -74,10 +74,9 @@
             var torrent = _this.model.get('torrent');
             var file = parseInt($(e.currentTarget).attr('data-file'));
             var actualIndex = parseInt($(e.currentTarget).attr('data-index'));
-            torrent.name = torrent.files[file].name;
 
             var torrentStart = new Backbone.Model({
-                torrent: torrent,
+                torrent: torrent.magnetURI,
```

Comment:
Butter will show the file selector for an http resource? Doesn't seem to make sense for me, but i'll test this cases too. Tks